### PR TITLE
Spanish translation

### LINKED ISSUE #18: Translate "Always display the current date in an Rmarkdown report" to Spanish
Always display the current date in an Rmarkdown report @oranwutang

_Originally posted by @oranwutang in https://github.com/thomas-neitmann/thomas-adventuRe/issues/12#issuecomment-774453383_

### PR DESCRIPTION
**Pull request by oranwutang**
Fixes #18 
Here is the translation.
I added the credit for the translation at the end of the document, but I guess that you may want to add it to the footer, so please review it.

I also corrected a little misspelling in the original file in English.

Kind regards

### CODE REVIEW

**Comment by oranwutang** [comment]
I have update the PR, please verify if the title of the post is correctly translated

File: content/posts/2020-03-22-rmarkdown-current-date.es.Rmarkdown [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: Mostrar la fecha actual automáticamente en un informe Rmarkdown
+author: Thomas Neitmann
+date: '2020-03-22'
+slug: rmarkdown-current-date
```
Comment (thomas-neitmann) [inline content/posts/2020-03-22-rmarkdown-current-date.es.Rmarkdown:5]: Could you translate this as well. This slug will be used as the URL of the page.

File: content/posts/2020-03-22-rmarkdown-current-date.es.Rmarkdown [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+`Rmarkdown` es una herramienta fantástica para crear informes impulsados por datos que requieran actualizarse regularmente. Para que los lectores sepan que el informe está realmente actualizado, es importante mostrar la fecha actual debajo del título.
+
+<!-- An obvious solution to display the current date is to update the date field in the YAML header manually before knitting. But this is both error prone and you are likely to forget about it. Instead you can dislay the current date in an automatic way using the method below. -->
+
+Una solución obvia para mostrar la fecha actual sería modificar manualmente el campo de la fecha en el encabezado YAML antes de compilar el informe (**knit**). Pero esto puede conllevar errores y es probable que se te olvide actualizarlo. En cambio, puedes mostrar la fecha actualizada automáticamente usando el método a continuación.
+
+```{r, eval=FALSE}
+---
+title: "Super Important Report"
```
Comment (thomas-neitmann) [inline content/posts/2020-03-22-rmarkdown-current-date.es.Rmarkdown:31]: Could you translate the `title` as well?

File: content/posts/2020-03-22-rmarkdown-current-date.es.Rmarkdown [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+Sin embargo, hay que tener cuidado con el formato de la fecha. `Sys.Date()` devuelve la fecha actual con el formato ISO 8601 (`yyyy-mm-dd`). Es probable que tus lectores no estén acostumbrados a este formato y, por lo tanto, quieras utilizar el formato que se usa en tu región.
+
+¿Cómo podemos hacerlo? Usando la función `format()`. He aquí algunos ejemplos.
+
+```{r}
+current_date <- Sys.Date()
+format(current_date, "%d.%m.%Y")
+format(current_date, "%d. %B %Y")
+format(current_date, "%m/%d/%Y")
```
Comment (thomas-neitmann) [inline content/posts/2020-03-22-rmarkdown-current-date.es.Rmarkdown:57]: Would it make sense to use a Spanish variable name?

File: content/posts/2020-03-22-rmarkdown-current-date.es.Rmarkdown [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+```{r, eval=FALSE}
+---
+title: "Super Important Report"
+author: "Thomas Neitmann"
+date: `r format(Sys.Date(), "%d. %B %Y")`
+---
+```
+
+Este artículo ha sido traducido del original en inglés por Gustavo Zapata Wainberg.
```
Comment (thomas-neitmann) [inline content/posts/2020-03-22-rmarkdown-current-date.es.Rmarkdown:72]: Would you like to include a link to your Twitter or elsewhere?

File: content/posts/2020-03-22-rmarkdown-current-date.markdown [hunk 5]
```diff
@@ -18,7 +18,7 @@ images: ~
 
 `Rmarkdown` is a great tool for creating data-driven reports that have to be updated on a regular basis. To communicate to stakeholders that the report is indeed up to date it's important to display the current date below the title.
 
-An obvious solution to display the current date is to update the date field in the YAML header manually before knitting. But this is both error prone and you are likely to forget about it. Instead you can dislay the current date in an automatic way using the method below.
+An obvious solution to display the current date is to update the date field in the YAML header manually before knitting. But this is both error prone and you are likely to forget about it. Instead you can display the current date in an automatic way using the method below.
```
Comment (thomas-neitmann) [inline content/posts/2020-03-22-rmarkdown-current-date.markdown:21]: Thanks for spotting this typo!

**Review by thomas-neitmann — CHANGES_REQUESTED** [review]
Thank you so much! Please have a look at my minor comments.

File: content/posts/2020-03-22-rmarkdown-current-date.es.Rmarkdown (hunk 2 above)
Comment (oranwutang) [reply content/posts/2020-03-22-rmarkdown-current-date.es.Rmarkdown]: Solved

File: content/posts/2020-03-22-rmarkdown-current-date.es.Rmarkdown (hunk 3 above)
Comment (oranwutang) [reply content/posts/2020-03-22-rmarkdown-current-date.es.Rmarkdown]: Surely it does!

File: content/posts/2020-03-22-rmarkdown-current-date.es.Rmarkdown (hunk 4 above)
Comment (oranwutang) [reply content/posts/2020-03-22-rmarkdown-current-date.es.Rmarkdown:72]: I think that it is enough with just the name

**Comment by oranwutang** [comment]
Hi Thomas, I have pushed the file with the queries solved. I think it is ready for your review.

**Review by thomas-neitmann — APPROVED** [review]
Muchas gracias!!

**Comment by oranwutang** [comment]
¡De nada, Thomas!

**Pull request merged**